docs/source/kap/overview.rst

```diff
@@ -76,7 +76,7 @@ between OP/OPAL/OPLIB can be found in `Farag et al. 2024 <https://doi.org/10.384
    Location of each Type-1 opacity table in the :math:`X–Z` plane (left panel) and the
    log:math:`T`–log:math:`R` plane (right panel). Orange circles mark the location of
    the 126 OPAL Type-1 tables (Rogers & Iglesias 1992). Blue circles
-   mark the location of the new 1194 Type-1 opacity tables (`Farag et al. 2024 <https://doi.org/10.3847/1538-4357/ad4355>`_).
```

Comment:
link in figure caption was not rendering, so I removed it because it is referenced in the paragraph above